pro_tes/utils/db.py

```diff
@@ -23,17 +23,17 @@ class DbDocumentConnector:
     """
 
     def __init__(
-        self,
-        collection: Collection,
-        worker_id: str,
+            self,
+            collection: Collection,  # type: ignore
+            worker_id: str,
```

Comment:
Why the double indenting (8 chars) on these? Did `black` do that? We should have only one level of indentation for the function signatures

Reply:
my pycharm by default reformats to 8 chars indent, let me fix that in the next commit.